core/document/src/operation.rs

```diff
@@ -1,3 +1,3 @@
 pub enum Operation {
-	AddCircle((f64, f64), f64),
+	AddCircle { cx: f64, cy: f64, r: f64 },
```

Comment:
Is this a dependency thing or why can't we just use `AddCircle(kurbo::Circle)`?

Reply:
We could debate on whether the use of kurbo should be an implementation detail, because we might replace kurbo with a more fully featured svg lib